### PR TITLE
Dynamic Simulation - Correct regression on event label

### PR DESCRIPTION
**Pull request by thangqp**
Regression from : https://github.com/gridsuite/gridstudy-app/pull/1921/files#diff-e67939fd25c650222db710f18764d10ae69454b0e8ad680f5e10177c2db93ceaL332-L342

After corrected
event_label_corrected

### CODE REVIEW

File: src/translations/dynamic/events-locale-fr.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const events_locale_fr = {
+    // Used in edition dialog and list of dynamic simulation events with format `Event{eventType}{equipmentType}`
+    EventDisconnectLine: 'Trip line {computedLabel}',
+    EventDisconnect2WTransformer: 'Trip transformer {computedLabel}',
+    EventDisconnect3WTransformer: 'Trip transformer {computedLabel}',
+    EventDisconnectLoad: 'Trip load {computedLabel}',
+    EventDisconnectGenerator: 'Trip generator {computedLabel}',
+    EventNodeFaultBus: 'Node fault on bus {computedLabel}',
+};
```
Comment (EstherDarkish) [inline src/translations/dynamic/events-locale-fr.ts:16]: This is not translated

**Review by EstherDarkish — CHANGES_REQUESTED** [review]

File: src/translations/dynamic/events-locale-fr.ts (hunk 1 above)
Comment (thangqp) [reply src/translations/dynamic/events-locale-fr.ts]: DONE

File: src/translations/dynamic/menu-locale-fr.ts [hunk 2]
```diff
@@ -29,6 +23,14 @@ const menu_locale_fr = {
     UnableToTrip2WTransformer: 'Impossible de déclencher le transformateur',
     UnableToTrip3WTransformer: 'Impossible de déclencher le transformateur',
     UnableToTripLine: 'Impossible de déclencher la ligne',
+
+    // Dynamic simulation events with format {eventType}{equipmentType}`
+    Disconnect2WTransformer: 'Déclencher le transformateur',
+    Disconnect3WTransformer: 'Déclencher le transformateur',
+    DisconnectGenerator: 'Déclencher le group',
```
Comment (EstherDarkish) [inline src/translations/dynamic/menu-locale-fr.ts:30]:
```suggestion
    DisconnectGenerator: 'Déclencher le groupe',
```

File: src/translations/dynamic/events-locale-fr.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const events_locale_fr = {
+    // Used in edition dialog and list of dynamic simulation events with format `Event{eventType}{equipmentType}`
+    EventDisconnect2WTransformer: 'Déclencher transformateur {computedLabel}',
+    EventDisconnect3WTransformer: 'Déclencher transformateur {computedLabel}',
+    EventDisconnectGenerator: 'Déclencher group {computedLabel}',
```
Comment (EstherDarkish) [inline src/translations/dynamic/events-locale-fr.ts:12]:
```suggestion
    EventDisconnectGenerator: 'Déclencher groupe {computedLabel}',
```

File: src/translations/dynamic/menu-locale-fr.ts (hunk 2 above)
Comment (thangqp) [reply src/translations/dynamic/menu-locale-fr.ts]: DONE

File: src/translations/dynamic/events-locale-fr.ts (hunk 3 above)
Comment (thangqp) [reply src/translations/dynamic/events-locale-fr.ts]: DONE

**Review by EstherDarkish — APPROVED** [review]
Code OK Test OK

**Pull request merged**